Formula/okteto.rb

```diff
@@ -0,0 +1,18 @@
+class Okteto < Formula
+  desc "Build better apps by developing and testing code directly in Kubernetes"
+  homepage "https://okteto.com"
+  url "https://github.com/okteto/okteto/archive/1.6.1.tar.gz"
+  sha256 "10e8ac27ec8d2e8d37d2eb779d71dcda66e1f5751ba84ed9d197ddc1b2fe039b"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-s -w -X github.com/okteto/okteto/pkg/config.VersionString=#{version}"
+    tags = "osusergo netgo static_build"
+    system "go", "build", "-o", "#{bin}/#{name}", "-trimpath", "-ldflags", ldflags, "-tags", tags
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/#{name} version")
```

Comment:
might be tests can be improved?

Reply:
It's a single statically linked binary and a simple CLI tool. Don't think there is a need for more in depth testing.

Reply:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
If you insist. Done.